tests/testthat/test-workflow-bbi.R

```diff
@@ -48,13 +48,18 @@ withr::with_options(list(rbabylon.bbi_exe_path = BBI_PATH,
   #######################
 
   test_that("step by step create_model to submit_model to model_summary works", {
+    # TODO: this test needs to clean up after itself by removing the files it
+    # created
+
     # create model spec
-    mod1 <- suppressWarnings(new_model(
-      .yaml_path = 1,
-      .description = ORIG_DESC,
-      .tags = ORIG_TAGS,
-      .bbi_args = list(overwrite = TRUE, threads = 4)
-    ))
+    mod1 <- suppressWarnings(
```

Comment:
this is only semi-related, but do we know what warnings this is intended to suppress? I generally don't think we should use `suppressWarnings` in tests (much preferring `suppressSpecificWarning` if we need to catch something unrelated to the test).

I only ask here because I'm wondering if it's related to behavior that changed in this PR.

Reply:
Good point. We can just remove this `suppressWarnings()` call.